Change link for JUL MDC and add dependency (#6563)
getsentry/sentry-docs: src/platforms/java/guides/jul/usage/advanced-usage.mdx

description: "Learn about adding extra data using the MDC system provided by SLF4j."
---

> When any SLF4J implementation supporting MDC is on the classpath (for example Logback, Log4j2, JUL), it’s possible to add extra data to events thanks to [the MDC system](https://www.slf4j.org/api/org/slf4j/helpers/BasicMDCAdapter.html).

Reviewer: ```suggestion
You can add extra data to events using [the MDC system](https://www.slf4j.org/api/org/slf4j/helpers/BasicMDCAdapter.html) for any SLF4J implementation supporting MDC that's on a classpath, (for example Logback, Log4j2, JUL).
```

When
